src/main/java/org/spongepowered/api/event/cause/EventContextKeys.java

```diff
@@ -45,7 +45,7 @@
  */
 public final class EventContextKeys {
 
-    public static final EventContextKey<UUID> CREATOR = createFor("CREATOR");
+    public static final EventContextKey<User> CREATOR = createFor("CREATOR");
```

Comment:
I am of the belief that we would be able to reduce the lookups throughout the system if the context contained a link to the user initially, I'm going to be doing the phase context refactors shortly after this branch is merged which should very well reduce the actual lookups throughout the game's life cycle. If we were to keep it as a `UUID`, plugins would have to consistently lookup the same profiles we're already looking up in the implementation.